Sources/ARSAL_Time.c

```diff
@@ -122,7 +163,12 @@ int ARSAL_Time_GetLocalTime (struct timespec *res, struct tm *localTime)
         /* No else --> Do nothing if res is NULL */
         if (localTime != NULL)
         {
+#ifdef _WIN32
+            struct tm *tmpLocalTime = localtime(&(ts.tv_sec));
```

Comment:
Right, I read it wrong (I think i am a bit rusty with win32 API....)

Both the 32-bit and 64-bit versions of gmtime, mktime, mkgmtime, and localtime all use a single tm structure per thread for the conversion. Each call to one of these routines destroys the result of the previous call.